contracts/DCAFeeManager/DCAFeeManager.sol

```diff
@@ -122,7 +143,7 @@ contract DCAFeeManager is Governable, IDCAFeeManager {
   }
 
   modifier onlyOwnerOrAllowed() {
-    if (!isGovernor(msg.sender) && !hasAccess[msg.sender]) revert CallerMustBeOwnerOrHaveAccess();
+    if (!hasAccess[msg.sender] && !isGovernor(msg.sender)) revert CallerMustBeOwnerOrHaveAccess();
```

Comment:
I figured that permissioned actions would be mostly triggered by allowed addresses, and not the governor. So by chaning the order, we can avoid a SLOAD